packages/design-system/src/images/usa-banner-flag.svg

```diff
@@ -1,4 +1,6 @@
-<svg width="16" height="11" xmlns="http://www.w3.org/2000/svg">
+<svg role="img" width="16" height="11" xmlns="http://www.w3.org/2000/svg" aria-labelledby="banner-flag-title banner-flag-description" focusable="false">
```

Comment:
Isnt `aria-labelledby` already handled elsewhere? The SVG file should be a generic SVG file, attributes that depend on ids and classes should be applied when they are actually used in React

Reply:
Update: sorry, I didn't realize the aria attributes are using `id`s that are also defined in the same SVG file. If that's the case then its fine